docs/installation/settings.rst

```diff
@@ -0,0 +1,7 @@
+***************
+Tethys Settings
```

Comment:
While I like the idea of this... it really could just be a link to the Django settings page, no? It also occurs to me that we may have a name conflict with the Tethys "settings" as in settings.py and Tethys "settings" as in settings for an app... Could generate some confusion.